sales_team_operating_unit/tests/test_crm_team_operating_unit.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from openerp.tests import common
+
+
+class TestSaleTeamOperatingUnit(common.TransactionCase):
+
+    def setUp(self):
+        super(TestSaleTeamOperatingUnit, self).setUp()
+        self.res_users_model = self.env['res.users']
+        self.crm_team_model = self.env['crm.team']
+        # Groups
+        self.grp_sale_mngr = self.env.ref('base.group_sale_manager')
+        self.grp_user = self.env.ref('base.group_user')
+        # Company
```

Comment:
There's no need for these comments. You can use significant variable names instead.